### PR TITLE
release post

### PR DESCRIPTION
**Pull request by gmauro**

### CODE REVIEW

**Comment by gmauro** [comment]
ping @beatrizserrano @bgruening

File: _posts/2021-03-16-galaxy-21-01.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [release]
+title: UseGalaxy.eu update to 21.01
+supporters:
+ - denbi
+ - elixir
+---
+
+UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:10]:
```suggestion
UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview!
```

File: _posts/2021-03-16-galaxy-21-01.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  and improved job creation performance for certain classes of tools and copying Histories, Datasets and Dataset Collections (`Pull Request 10539`_, `Pull Request 10549`_).
+  The performance of reading tool wrappers when starting Galaxy has been much improved. We have therefore decided to disable the tool document cache by default,
+  since it only provides negligible speedups (`Pull Request 10804`_, `Pull Request 11447`_). The tool document cache can be re-enabled with the `enable_tool_document_cache <https://github.com/galaxyproject/galaxy/blob/283ec1772fcc0e574997dc2c5ad12dc86e3e1d37/lib/galaxy/config/sample/galaxy.yml.sample#L626>`_ setting.
+
+### Offload Zip Archive Creation to NGINX
+  Galaxy can now utilize the NGINX module `mod_zip <https://www.nginx.com/resources/wiki/modules/zip/>`__ to assemble and compress dataset collection and library dataset archives on the fly.
+  This method is more efficient and can create large archives without affecting the Galaxy server process. 
+
+### Workflow Enhancements and Fixes
+  This release contains many new features, enhancements and bug fixes for creating, running and reporting workflows
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:29]:
```suggestion
  This release contains many new features, enhancements and bug fixes for creating, running and reporting workflows.
```

File: _posts/2021-03-16-galaxy-21-01.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  
+### History Import and Export
+  Histories can now be imported from and exported to the pluggable file source architecture introduced in the 20.09 release of Galaxy.
+  This includes the users' FTP folder, S3, Dropbox, ownCloud etc. In addition the History Export and Import dialogues have been rewritten
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:33]:
```suggestion
  This includes the users' FTP folder, S3, Dropbox, ownCloud, etc. In addition, the History Export and Import dialogues have been rewritten
```

File: _posts/2021-03-16-galaxy-21-01.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  to provide access to these new options and to provide more feedback during Import and Export process.
+
+
+---
+
+If you find issues, please [let us know](mailto:galaxy@informatik.uni-freiburg.de)!
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:39]:
```suggestion
If you find issues, please [let us know](mailto:contact@informatik.uni-freiburg.de)!
```

File: _posts/2021-03-16-galaxy-21-01.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+ - elixir
+---
+
+UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview
+
+
+## Some Highlights
+
+### New History Panel (Beta)
+  This release includes a preview of the new History Panel. The new history menu can be enabled and disabled in the options menu of the History Panel.
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:16]:
```suggestion
  This release includes a preview of the new History Panel. The new history panel can be enabled and disabled in the options menu -> Use Beta History Panel.
```

File: _posts/2021-03-16-galaxy-21-01.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+### Performance Improvements
+  This release includes many performance improvements for browsing and searching Data libraries (Many thanks to `Oleg Zharkov <https://github.com/OlegZharkov>`__, `Pull Request 10204`_, `Pull Request 11099`_, `Pull Request 10557`_, `Pull Request 10558`_, `Pull Request 10807`_),
+  and improved job creation performance for certain classes of tools and copying Histories, Datasets and Dataset Collections (`Pull Request 10539`_, `Pull Request 10549`_).
+  The performance of reading tool wrappers when starting Galaxy has been much improved. We have therefore decided to disable the tool document cache by default,
+  since it only provides negligible speedups (`Pull Request 10804`_, `Pull Request 11447`_). The tool document cache can be re-enabled with the `enable_tool_document_cache <https://github.com/galaxyproject/galaxy/blob/283ec1772fcc0e574997dc2c5ad12dc86e3e1d37/lib/galaxy/config/sample/galaxy.yml.sample#L626>`_ setting.
+
+### Offload Zip Archive Creation to NGINX
```
Comment (bgruening) [inline _posts/2021-03-16-galaxy-21-01.md:24]: this is too technical I think for the users. Do we use this actually?

Comment (gmauro) [reply _posts/2021-03-16-galaxy-21-01.md:24]: I agree, we can remove it.

File: _posts/2021-03-16-galaxy-21-01.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview!
+
+
+## Some Highlights
+
+### New History Panel (Beta)
+  This release includes a preview of the new History Panel. The new history panel can be enabled and disabled in the options menu -> Use Beta History Panel.
+
+### Performance Improvements
+  This release includes many performance improvements for browsing and searching Data libraries (Many thanks to `Oleg Zharkov <https://github.com/OlegZharkov>`__, `Pull Request 10204`_, `Pull Request 11099`_, `Pull Request 10557`_, `Pull Request 10558`_, `Pull Request 10807`_),
```
Comment (beatrizserrano) [inline _posts/2021-03-16-galaxy-21-01.md:19]: not sure what's going on here, but it's displayed in a strange way, might be the double _?

Comment (gmauro) [reply _posts/2021-03-16-galaxy-21-01.md:19]: could be. let me fix it

Comment (gmauro) [reply _posts/2021-03-16-galaxy-21-01.md]: I don't have yet the build environment in the new notebook. Can you try again?

Comment (bgruening) [reply _posts/2021-03-16-galaxy-21-01.md]: Isn't that RST syntax and we are here in Markdown?

**Comment by bgruening** [comment]
https://docs.galaxyproject.org/en/release_21.01/releases/21.01_announce_user.html we can link that.

I can work on that in the evening if its not urgent.

File: _posts/2021-03-16-galaxy-21-01.md (hunk 7 above)
Comment (beatrizserrano) [reply _posts/2021-03-16-galaxy-21-01.md]: looks good now, at least in the GitHub interface :)

**Comment by beatrizserrano** [comment]
ready to merge? :)

**Comment by beatrizserrano** [comment]
> https://docs.galaxyproject.org/en/release_21.01/releases/21.01_announce_user.html we can link that.
> 
> I can work on that in the evening if its not urgent.

Why are we not using these more user-friendly notes directly? Do we expect devs to look at the post?

**Comment by gmauro** [comment]
This morning when I started the post, the release notes for users weren't available (or at least I wasn't able to found them).
They seem also to me, more appealing.

**Comment by bgruening** [comment]
Thanks all!

**Pull request merged**